### PR TITLE
Update shields to match web

### PR DESCRIPTION
**Pull request by BillCarsonFr**
Updated shield drawable to match those on wed (see https://www.figma.com/file/X4XTH9iS2KGJ2wFKDqkyed/Compound?node-id=1373%3A13)
Fixes https://github.com/vector-im/element-meta/issues/52

There is still a difference though, the icons have a white border, but looks like on web they don't (see verification conclusion in dark theme on web)

image

### CODE REVIEW

File: vector/src/main/java/im/vector/app/features/crypto/verification/epoxy/BottomSheetVerificationBigImageItem.kt [hunk 1]
```diff
@@ -35,10 +35,23 @@ abstract class BottomSheetVerificationBigImageItem : VectorEpoxyModel<BottomShee
 
     override fun bind(holder: Holder) {
         super.bind(holder)
-        holder.image.render(roomEncryptionTrustLevel)
+        when (roomEncryptionTrustLevel) {
```
Comment (bmarty) [inline vector/src/main/java/im/vector/app/features/crypto/verification/epoxy/BottomSheetVerificationBigImageItem.kt:38]: I would prefer to keep the old code and add a optional parameter in the `ShieldImageView` fun `render(..., noBorder: Boolean = false)` to use alternative image in the case the parameter is true.
Can you do that please?

File: vector/src/main/res/layout/item_verification_big_image.xml [hunk 2]
```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<im.vector.app.core.ui.views.ShieldImageView xmlns:android="http://schemas.android.com/apk/res/android"
+<ImageView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:contentDescription="@string/trusted"
```
Comment (bmarty) [inline vector/src/main/res/layout/item_verification_big_image.xml:4]: `android:contentDescription` -> `tools:contentDescription`

Also please format the file.

File: vector/src/main/res/layout/item_verification_big_image.xml [hunk 3]
```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<im.vector.app.core.ui.views.ShieldImageView xmlns:android="http://schemas.android.com/apk/res/android"
+<ImageView xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (bmarty) [inline vector/src/main/res/layout/item_verification_big_image.xml:2]: So you can revert the change on this line too

**Review by bmarty — COMMENTED** [review]
Just a small remark on the impl, else LGTM

File: vector/src/main/java/im/vector/app/features/crypto/verification/epoxy/BottomSheetVerificationBigImageItem.kt (hunk 1 above)
Comment (BillCarsonFr) [reply vector/src/main/java/im/vector/app/features/crypto/verification/epoxy/BottomSheetVerificationBigImageItem.kt:38]: done

File: vector/src/main/res/layout/item_verification_big_image.xml (hunk 2 above)
Comment (BillCarsonFr) [reply vector/src/main/res/layout/item_verification_big_image.xml:4]: done

File: vector/src/main/res/layout/item_verification_big_image.xml (hunk 3 above)
Comment (BillCarsonFr) [reply vector/src/main/res/layout/item_verification_big_image.xml:2]: done

File: vector/src/main/java/im/vector/app/features/crypto/verification/epoxy/BottomSheetVerificationBigImageItem.kt [hunk 4]
```diff
@@ -35,7 +35,7 @@ abstract class BottomSheetVerificationBigImageItem : VectorEpoxyModel<BottomShee
 
     override fun bind(holder: Holder) {
         super.bind(holder)
-        holder.image.render(roomEncryptionTrustLevel)
+        holder.image.render(roomEncryptionTrustLevel, borderLess = true)
```
Comment (bmarty) [inline vector/src/main/java/im/vector/app/features/crypto/verification/epoxy/BottomSheetVerificationBigImageItem.kt:38]: nice!

**Review by bmarty — APPROVED** [review]
LGTM, I've double check all the vector drawables, and everything is fine

**Comment by chagai95** [comment]
Is there a plan to have this as a config?
Unfortunately, we have another color theme...

File: vector/src/main/java/im/vector/app/core/ui/views/ShieldImageView.kt [hunk 5]
```diff
@@ -36,21 +36,30 @@ class ShieldImageView @JvmOverloads constructor(
...
 
-    fun render(roomEncryptionTrustLevel: RoomEncryptionTrustLevel?) {
+    fun render(roomEncryptionTrustLevel: RoomEncryptionTrustLevel?, borderLess: Boolean = false) {
         isVisible = roomEncryptionTrustLevel != null
 
         when (roomEncryptionTrustLevel) {
             RoomEncryptionTrustLevel.Default -> {
                 contentDescription = context.getString(R.string.a11y_trust_level_default)
-                setImageResource(R.drawable.ic_shield_black)
+                setImageResource(
```
Comment (ouchadam) [inline vector/src/main/java/im/vector/app/core/ui/views/ShieldImageView.kt:45]: @chagai95 updating these to use [setAttributeTintedImageResource](https://github.com/vector-im/element-android/blob/develop/vector/src/main/java/im/vector/app/core/extensions/ViewExtensions.kt#L73) and then supplying a theme colour will allow customisations
